.travis/smoke-test.sh

```diff
@@ -8,3 +8,6 @@ function finish() {
 trap finish EXIT
 
 make start wait-healthy
+
+ping=$(curl -sS 'http://localhost:8080/' 2>&1)
+[[ $ping == 'OK' ]]
```

Comment:
is that the response body? Any value is good

Reply:
Yeah, Koa's default when nothing is set.

Reply:
https://github.com/libero/article-store/pull/3/files#diff-f98485da1849ae631ba5c04e5cec07dd seems to say this is the response status?

Reply:
Koa sets the body based on the status code.

Reply:
Might be we should just test the code.

Reply:
Possibly even redundant as it waits for the containers to be healthy? 🤔

Reply:
Reachable from the host is more than container being healthy as it tests also ports and how they are forwarded. Good as it is